types/cassanknex/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"space-before-function-paren": [true, "always"],
```

Comment:
Can you turn off these lint rule exceptions so that this file is just an extension to the main lint rules?